Stripe/STPAPIClient.m

```diff
@@ -621,3 +623,25 @@ - (void)confirmPaymentIntentWithParams:(STPPaymentIntentParams *)paymentIntentPa
 }
 
 @end
+
+#pragma mark - Payment Methods
+
+@implementation STPAPIClient (PaymentMethods)
+
+- (void)createPaymentMethodWithParams:(STPPaymentMethodParams *)paymentMethodParams
```

Comment:
will you also add a version with just a payment method id?

Reply:
that would just be a "retrieve" not "create" right? but yes, +1 to also adding

Reply:
My current understanding is that we won't support 'retrieve' for now (https://stripe.slack.com/archives/C5522BZPS/p1551915861432300)

Reply:
oh, sorry, i missed that. did we already build it for android? if yes, it seems unfortunate to have that gap (but if there's nothing we can do about it, that's ok)

Reply:
Android has retrieve PaymentIntent

Reply:
I'll add this to my TODO list, and add in a later PR.